frontend/app/settings/page.tsx

```
@@ -28,51 +35,222 @@ const DetailsSection = () => {
+                  className="rounded"
+                />
+              </div>
+            )}
+            <div>
+              <span className="font-medium">{currentCompany ? currentCompany.name : "Current Workspace"}</span>
+              {contractorStatusQuery.data?.contractorData && (
+                <p className="text-sm text-gray-500">
+                  Active contractor since{" "}
+                  {new Date(contractorStatusQuery.data.contractorData.createdAt).toLocaleDateString()}
+                </p>
+              )}
```

ESLint

[error] Potential leaked value that might cause unintentionally rendered values or rendering crashes

(react/jsx-no-leaked-render)